### PR TITLE
Greenfield Project Quarterly Delivery 

### PR DESCRIPTION
**Pull request by jaysonmc**
Adding content into one of the bullet points under the citizen perspective

### CODE REVIEW

File: _pages/en/it-picture-medium-term.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ In the following sections, we're trying to take the same end state and create "v
 
 ## IITB-ESDC
 
-- IITB controls 20% of it's funding for Internal IT Projects 
+- IITB strictly controls 20% of it's funding for Internal IT Projects 
 - IITB is involved in business discussions and decisions (TODO: Are we now? What body should we be apart of, and who should be a part of it?)
 - 75% of employees are equipped with mobile devices allowing them to work from anywhere (TODO: Map trend and profiles on to metric )
```
Comment (remyb2canada) [inline _pages/en/it-picture-medium-term.md]: We're talking ESDC employees or IITB only?

File: _pages/en/it-picture-medium-term.md [hunk 2]
```diff
@@ -30,7 +30,34 @@ In the following sections, we're trying to take the same end state and create "v
 - Cloud First
```
Comment (remyb2canada) [inline _pages/en/it-picture-medium-term.md]: I'd specify *public* cloud first

File: _pages/en/it-picture-medium-term.md [hunk 3]
```diff
@@ -30,7 +30,34 @@ In the following sections, we're trying to take the same end state and create "v
...
+![alt text](../../assets/images/aspectOfSoftwareDeliveryPerformance.PNG)
+
+Of respondents 
+
+- 7% were Elite Performers
+- 41% were High Performers
+- 37% were Medium Performers
+- 15% were Low Performers 
+
+Given that low performers are able to deploy between once per week and once per month, within 5 years IITB will must be able to deploy into production within 4 months within 5 years for all new projects (including those replacing legacy systems).
```
Comment (remyb2canada) [inline _pages/en/it-picture-medium-term.md]: reword:
within 5 years IITB will must be able to deploy into production within 4 months within 5 years for all new projects 
to
within 5 years IITB will must be able to deploy into production within 4 months for all new projects

File: _pages/en/it-picture-medium-term.md [hunk 4]
```diff
@@ -30,7 +30,34 @@ In the following sections, we're trying to take the same end state and create "v
...
+
+A world of opportunities awaits IITB in our long term plan if we are able to realize the benefits of treading the path that has already been laid before us by the high performers and world leaders from the IT community. Without it, in the world of running 10 year projects our 5 year trajectory is easily predictable -- the organization will look identical to the organization that exists today.
```
Comment (remyb2canada) [inline _pages/en/it-picture-medium-term.md]: link to the it-picture-long-term.md

File: _pages/en/it-picture-medium-term.md (hunk 1 above)
Comment (jaysonmc) [reply _pages/en/it-picture-medium-term.md]: Well, I didn't add that in this pull request but I agree. That's why we have the TODO marker there. That needs to be researched and nuanced. 

We need to reference literature that allows for us to say that, and add profiles and nuance the statement

File: _pages/en/it-picture-medium-term.md (hunk 3 above)
Comment (jaysonmc) [reply _pages/en/it-picture-medium-term.md]: Made the recommended changes

**Comment by jaysonmc** [comment]
Added in the sections recommended by @smellems

File: _pages/en/it-picture-medium-term.md (hunk 2 above)
Comment (jaysonmc) [reply _pages/en/it-picture-medium-term.md]: Changed to 'public cloud first'

**Pull request merged**